<launch>
	<!-- Start VRPN Client node: getting data from Optitrack -->
	<!-- Include  the file that contains your custom vrpn launch-->
	<include file="$(find tic_tac_drone)/launch/across.launch"/>

	<!-- Start Optitrack node: calculating and distributing postion data -->
	<node pkg="tic_tac_drone" type="OptitrackNode.py" name="Optitrack">
		<remap from="MyUAV/pose" to="vrpn_client_node/UAV1/pose"/>
		<remap from="OpUAV/pose" to="vrpn_client_node/UAV2/pose"/>
		<remap from="Field/pose" to="vrpn_client_node/Field/pose"/>
	</node>

	<!-- Start Game detection node: planning course and detecting played move -->
	<node pkg="tic_tac_drone" type="GameDetectionNode.py" name="GameDetection" output="screen">
		<param name="_cell_size" value="0.5"/>
		<param name="_max_height" value="1"/>
	</node>
</launch>